### ISSUE TITLE
Controlgroup button height

### ISSUE THREAD

**Issue by vrbvillor** [jquery/jquery-mobile#8503]
Just as the photo.
![qq 20160812175235](https://cloud.githubusercontent.com/assets/8016761/18045833/768f1ec4-6e07-11e6-833a-bd0feca0f494.png)

Such as http://demos.jquerymobile.com/1.4.5/forms-field-contain/  and other ones. In webkit browser . That seems it always happens when using `data-mini="true"`.

**Comment by jaspermdegroot** [comment]
@vrbvillor 

This has been fixed in branch master. See http://view.jquerymobile.com/master/demos/forms-field-contain/ for example.

**Comment by vrbvillor** [comment]
But this page has new bugs. Look at the pics below.

![qq 20160812175235](https://cloud.githubusercontent.com/assets/8016761/18114665/890b7b5a-6f6b-11e6-9242-b7433cd0f92b.png)
![qq 20160831110916](https://cloud.githubusercontent.com/assets/8016761/18114667/8912eaf2-6f6b-11e6-9d68-a3675dbfc7e7.png)
![qq 20160831110925](https://cloud.githubusercontent.com/assets/8016761/18114669/891454dc-6f6b-11e6-8089-433fcc090914.png)
![qq 20160831110943](https://cloud.githubusercontent.com/assets/8016761/18114668/89130c62-6f6b-11e6-9309-63bd4bd7644d.png)
![qq 20160831110954](https://cloud.githubusercontent.com/assets/8016761/18114666/89129bb0-6f6b-11e6-9bde-0581d99eae0c.png)

**Comment by jaspermdegroot** [comment]
@vrbvillor

> The third component is on the new line. But the content of the select is not too long to have this apperance.

There is already a ticket for this: #8493

> What's the select for? To toggle switch the slider on/off ??

The flip toggle examples have been fixed in PR #8494 (preview: http://view.jquerymobile.com/demos/demos/forms-field-contain/), but this PR has not been merged in master yet.

> Why the caret of the two select are not at the same position?

We were aware of this problem, but there was no ticket for this so I opened one: #8506 

> This one the button without icon is not as high as the one with icon.
But in this picture, it seems the left one is over hight.

I will look into this. Can you tell me what font is used in your examples?

**Comment by vrbvillor** [comment]
> This one the button without icon is not as high as the one with icon. But in this picture, it seems the left one is over hight.   

That is not on my page, that's on your website. If the problem is due to the font family, um...... the F12 calculated the family is Sans Serif.

**Comment by jaspermdegroot** [comment]
@vrbvillor 

I know that it's our demo page. I just want to test with the same font as you.
What Windows version and what version of IE/Edge are you using?

**Comment by vrbvillor** [comment]
win7 and webkit browser.

**Comment by jaspermdegroot** [comment]
@vrbvillor 

Your system uses the SimSun font for latin characters. I tested on a Windows 7 system with that font and was able to reproduce the problem. I tested with many other serif and sans-serif fonts but it only happened with SimSun.

In the code inspector in Chrome, can you change the font-family in this rule:
```
body,
input,
select,
textarea,
button,
.ui-button {
	font-size: 1em;
	line-height: 1.3;
	font-family: sans-serif /*{global-font-family}*/;
}
```
to:
```
font-family: Arial, sans-serif /*{global-font-family}*/;
```

Is the Arial font used now (and the problem gone) or still see the same font?

**Comment by vrbvillor** [comment]
The button height problem is gone. Excellent

**Comment by jaspermdegroot** [comment]
After some more testing I noticed that it didn't only happen with the SimSun font and found out that the cause of the problem is the inline block icon and how we adjust its vertical position.

**Comment by jaspermdegroot** [comment]
With commit https://github.com/jquery/jquery-mobile/commit/e6b36ecbd39e71869682d8b4323c5184da172178 we reduced the font-family to simply `sans-serif` to fix #5778. The idea was that every OS would use the default sans-serif font which in most cases is also the system font.

A better way to do is, is setting the font-family like this:
```font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", Arial, sans-serif```

This results in:
`-apple-system` for Safari iOS, Safari Mac OS, and Firefox Mac OS
`BlinkMacSystemFont` for Chrome Mac OS
`Segoe UI` for Windows
`Roboto` for Android and Chrome OS
`Oxygen-Sans` for KDE
`Ubuntu` for Ubuntu
`Cantarell` for GNOME
`Helvetica Neue` for old Mac OS
`Arial` for old Windows
`sans-serif` fallback

This will also prevent Chinese/Japanese/Korean Windows using SimSun.
